Kick-off clarifications (#105)
RoboCup-SPL/Rules: rules/game_process.tex

For kick-off, the robots run through three states: \texttt{ready}, \texttt{set}, and \texttt{playing}.

In the \texttt{ready} state, the robots must walk to legal kick-off positions for their team.
Upon reaching the \texttt{set} state, the robots must not move until the kick-off is signaled by the referee (\cf~\cref{sec:robot_states,sec:motion_in_set}).
The following restrictions apply during the \texttt{set} state until the ball becomes free:
\begin{itemize}
  \item The kick-off team can be positioned within their own half and the center circle.

> No robot may block ball placement by standing on the center mark.

Reviewer: ```suggestion
No robot may block the center mark.
```

> Only one robot of the kick-off team is allowed to stay outside the own half.

Reviewer: ```suggestion
Only one robot of the kick-off team is allowed to stay outside the own half of the center circle.
```

All robots outside the own half, except the closest to the center mark, are illegally positioned.
If the kick-off team has a robot positioned outside the own half, only this robot, called kick-off player, is allowed to execute the kick-off.

> If a different robot of the kick-off team is executing the kick-off, this robot is considered to be illegally positioned and a moved ball gets moved back.

Reviewer: ```suggestion
If a different robot of the kick-off team executes the kick-off, this robot is considered to be illegally positioned and a moved ball gets placed back.
```

  \item The defending team can be positioned within their own half excluding the center circle.
  \item Robots are allowed to be positioned inside their own goal. All other positions that are fully on the border strip are illegal for both teams.
  \item Both teams are also subject to restrictions on the goal area (\cf~\cref{sec:ip_own_goal_area}).
\end{itemize}

> All robots that do not reach legal positions or enter illegal positions before the ball is in play will be penalized with the ``Illegal Position'' penalty~(\cf \cref{sec:ip_kick_off}).

Reviewer: ```suggestion
All robots that do not reach legal positions before ``Set'' starts or enter illegal positions before the ball is in play will be penalized with the ``Illegal Position'' penalty~(\cf \cref{sec:ip_kick_off}).
```

> A referee places the ball on the center mark. If the ball is moved by one of the robots during \texttt{set}, it is moved back by one of the referees.

Reviewer: Dublicated to line 168
